src/builder.ts

```diff
@@ -1,9 +1,10 @@
 import axios from "axios";
-import { BuildableRunnable, AuthenticatedRunnable } from "./types/runnable";
+import { Runnable, BuildableRunnable, BuildableTemplate, AuthenticatedRunnable } from "./types/runnable";
```

Comment:
I think references to Runnable should be renamed to Module.

Reply:
Yeah I was gonna do that separately to not clutter up the diff with renamings, keep it for actual logic changes as much as possible.